### DISCUSSION TITLE
Rename Loopback to Offline and hide in-game info [fixed]

### DISCUSSION THREAD

**Discussion by ikonomov** [diasurgical/DevilutionX#6298] [Ideas]
Renaming Loopback to Offline (or something else) could help make the menu option a little more direct and clear to the end user.  Also the in-game info overlay for Loopback doesn't need a Password info and possibly the name of the game as well.

**Comment by AJenbo** [comment]
just make the pr

**Reply by ikonomov** [reply]
https://github.com/diasurgical/devilutionX/pull/6301/ I could use some help :)

**Comment by ikonomov** [comment]
fixed in https://github.com/diasurgical/devilutionX/pull/6300
